mlflow/models/evaluation/base.py

```diff
@@ -1854,24 +1652,46 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
                 custom_metrics=custom_metrics,
                 extra_metrics=extra_metrics,
                 custom_artifacts=custom_artifacts,
-                baseline_model=baseline_model,
                 predictions=predictions_expected_in_model_output,
             )
         finally:
             if isinstance(model, _ServedPyFuncModel):
                 os.kill(model.pid, signal.SIGTERM)
-            if isinstance(baseline_model, _ServedPyFuncModel):
-                os.kill(baseline_model.pid, signal.SIGTERM)
 
-        if not validation_thresholds:
-            return evaluate_result
+    # TODO: Remove this block in a future release when we
+    # remove the deprecated arguments.
+    if baseline_model is not None and validation_thresholds is not None:
+        from mlflow.models.evaluation.validation import validate_evaluation_results
+
+        warnings.warn(
+            "Model validation functionality is moved from `mlflow.evaluate` to the "
+            "`mlflow.evaluation.validate_evaluation_results()` API. The "
+            "`baseline_model` argument will be removed in the future release.",
```

Comment:
```suggestion
            "`baseline_model` argument will be removed in a future release.",
```

nit

Reply:
"the future release" sounds like we've already decided which release we'll remove baseline_model. `a future release` is like "somewhere in the future".

Reply:
ah didn't think much about it but that makes sense!